Automattic/wp-calypso
# Add domain transfer to any user form for domain only sites

Fixes https://github.com/Automattic/dotcom-forge/issues/3596

## Proposed Changes

* Add domain transfer to any user form for domain only sites

## Testing Instructions

Requires flag (applied by default in development env): `http://calypso.localhost:3000/domains/manage/all/test345678.blog/transfer/test345678.blog?flags=domains/transfer-to-any-user`

* Domain only site
* Click to transfer flow
* Should see new transfer to wordpress.com user option card
* Clicking through you should get the new transfer form
* Entering an email and clicking save should store the email in a blog option per D120926-code (requires patch applied)



![Screenshot 2023-09-07 at 14-23-28 test345678 blog — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/6eab8698-5674-4e22-b5f9-fcc140791cf0)
